modules/xmpp/SDPUtil.js

```diff
@@ -443,6 +443,30 @@ var SDPUtil = {
             .split(" ")
             .map(ssrcStr => parseInt(ssrcStr));
     },
+
+    /**
+     * Sets the given codecName as the preferred codec by
+     *  moving it to the beginning of the payload types
+     *  list (modifies the given mline in place)
+     * @param {object} videoMLine the video mline object from
+     *  an sdp as parsed by transform.parse
+     * @param {string} the name of the preferred codec
+     */
+    preferVideoCodec: function(videoMLine, codecName) {
```

Comment:
mixing let/const in a file is one thing but, in my opinion, mixing function definitions doesn't seem like a good idea.  even in projects that aren't consistent across an entire codebase, usually it is kept consistent at least within a file.  if the team disagrees i'll change but to me it seems like consistency for function definitions would trump the incremental amount of work we'd save whenever we go through and change this file over to es6.